cilium/cmd/version.go

```diff
@@ -18,18 +18,42 @@ import (
 	"fmt"
 
 	"github.com/cilium/cilium/pkg/version"
-
```

Comment:
Hi Steven, Thanks for helping out! Could you put this line back.
We generally follow this convention

standard packages

cilium packages

third party packages